site_scons/compiler_setup.py

```diff
@@ -53,6 +53,9 @@ def _base_setup(env):
                         '-fpic'])
 
     env.AppendIfSupported(CCFLAGS=DESIRED_FLAGS)
+    env.AppendIfSupported(LINKFLAGS=DESIRED_FLAGS)
+    if '-fstack-protector-strong' in env["CCFLAGS"]:
+        env.AppendENVPath("CGO_LDFLAGS", "-fstack-protector-strong", sep=" ")
```

Comment:
Please advise. This aims to solve a problem I was seeing on Frontera during link (for several objects):
```
undefined reference to `__stack_chk_guard'
```
Which is because the compile flags use `-fstack-protector-strong` but the link flags were not using the same. I don't know why this issue hasn't come up anywhere else, but this change resolves it on Frontera.